### ISSUE TITLE
[Feature Request] Add a `new_url` parameter to `render_table` macro to direct to a route for creating new records/items

### ISSUE THREAD

**Issue by caffeinatedMike** [greyli/bootstrap-flask#128]
It would be great if the `render_table` macro had an additional parameter for a link that would lead to the route that creates new records that are shown in the table. I imagine this would be best suited as a button or anchor rendered either in the table header row (maybe inside the "Actions" td?) or next to the table title.

**Example usages:**
```
    {{ render_table(
            data,
            titles=titles,
            show_actions=True,
            new_url=url_for('add_account'),
            edit_url=url_for('edit_account', account_id=':primary_key'),
            delete_url=url_for('delete_account', account_id=':primary_key')
     ) }}
```

Thoughts?

**Comment by greyli** [comment]
Yes, it will be useful! I will try to implement it next week (PR is welcome).

**Comment by caffeinatedMike** [comment]
@greyli Awesome, thanks! If I have time toward the tail-end of this week I plan to try helping with a PR to allow custom actions since it's something I'm in need of and noticed it in the list of TODOs in #50 .

I also noticed another item in that list 
> "Improve the display of table action buttons (the delete icon is not in line with others)."

I was able to easily fix the alignment (or at least improve it) with a single line of css:
```css
td > form > input[type="image"][alt="Delete"] {
    /* Fixes alignment of inline buttons in Actions column of tables */
    vertical-align: middle;
}
```
**Result:**
![2021-05-18 08_56_32-The Portal Portal](https://user-images.githubusercontent.com/22151742/118655078-34f06500-b7b7-11eb-81af-ab78d19a70a9.png)

**Comment by greyli** [comment]
Hi @caffeinatedMike, the "new_url" feature was implemented in #133, check out the screenshot in that PR. I will make a new release tomorrow.

> f I have time toward the tail-end of this week I plan to try helping with a PR to allow custom actions

Feel free to make a PR.

BTW, the delete icon issue was fixed in #132.

**Comment by caffeinatedMike** [comment]
@greyli that's great news! The custom actions addition is at the top of my list for next week.
